packages/sops/go-sockaddr.diff

```diff
@@ -0,0 +1,40 @@
+diff --git a/route_info_android.go b/route_info_android.go
+index 9885915a..94c25789 100644
+--- a/route_info_android.go
++++ b/route_info_android.go
+@@ -5,10 +5,6 @@ import (
+ 	"os/exec"
+ )
+
+-type routeInfo struct {
+-	cmds map[string][]string
+-}
+-
+ // NewRouteInfo returns a Android-specific implementation of the RouteInfo
+ // interface.
+ func NewRouteInfo() (routeInfo, error) {
+diff --git a/route_info_default.go b/route_info_default.go
+index 6df864ba..db0052fa 100644
+--- a/route_info_default.go
++++ b/route_info_default.go
+@@ -1,5 +1,5 @@
+-//go:build android || nacl || plan9 || js
+-// +build android nacl plan9 js
++//go:build nacl || plan9 || js
```

Comment:
This hunk doesn't look like it belongs in the patch since it's replacing a comment with a comment.

Reply:
See https://pkg.go.dev/cmd/go#hdr-Build_constraints. I don't know why route_info_default.go is always compiled.

Reply:
Comments that aren't.
Wonderful.